src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body,
+html {
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  max-width: 1200px;
+  box-sizing: border-box;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.link__home {
+  padding: 10px 0 6px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  align-content: center;
+  width: 641px;
```

Comment:
Same here. I don't think that it's a good idea to hardcore this value. Just add the correct margins between nav items